crates/cli/src/config/runtime.rs

```diff
@@ -120,6 +125,13 @@ pub fn runtime(args: &ArgMatches) -> Result<RuntimeConfig> {
 			return fut;
 		}
 
+		let tags: Vec<_> = action.events.iter().flat_map(|e| e.tags.clone()).collect();
```

Comment:
simpler to do a `action.events.find(|e| e.tags.contains...)` or something like it

Reply:
Good point. I've updated that.